src/popups/InfoFPWPopup/InfoFPWPopup.js

```diff
@@ -0,0 +1,114 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Feature from 'ol/Feature';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { compose } from 'lodash/fp';
+import Button from 'react-spatial/components/Button';
+import { IoIosArrowRoundBack, IoIosArrowRoundForward } from 'react-icons/io';
+
+import './InfoFPWPopup.scss';
+
+const propTypes = {
+  feature: PropTypes.instanceOf(Feature).isRequired,
+  language: PropTypes.string.isRequired,
+  t: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+class InfoFPWPopup extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imageIndex: 0,
+    };
+  }
+
+  setImageIndex(index) {
+    this.setState({
+      imageIndex: index,
+    });
+  }
+
+  render() {
+    const { feature, language, t } = this.props;
+    const { imageIndex } = this.state;
+
+    let text;
+    let image;
+    let pagination;
+
+    const pureText = feature.get(`${language}_text`) || feature.get('de_text');
+    if (pureText) {
+      // eslint-disable-next-line react/no-danger
+      text = <div dangerouslySetInnerHTML={{ __html: pureText }} />;
```

Comment:
As this will be a private topic, we could dare to use it. What do you think?
Currently, the customer can generate HTML code in Cartaro with <strong>, <br />, <ul>, <ol>, <li>, and different kind of <a>. On production, all of these are used.

Reply:
@oterral What do you think?

Reply:
Fine for me